about.html

```diff
@@ -0,0 +1,766 @@
+<!DOCTYPE html>
+<html lang="en" class="no-js" itemscope itemtype="http://schema.org/Article">
+    <!-- Begin Head -->
+    <head>        <!-- Basic -->
+        <title>Mozilla Kerala</title>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        
+        <meta name="Mozilla Kerala" content="Mozilla Kerala">
+        <meta name="copyright" content="All contents avaliable under creative commons licence">
+        <meta name="keywords" content="mozilla,mozillakerala,free,opensource,volnteer,webmaker,hosting">
+ 
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
I hope it would be good to load the script tags at the end of the page as it effects the time for the initial render.